src/states/GameState.hpp

```diff
@@ -9,6 +9,9 @@
 class GameState : public BaseState {
 private:
     BoardManager board;
+    TextureManager textures;
+
+    std::string background_image = "assets/images/background.jpg";
```

Comment:
Probably could be a constant in `constants.hpp` instead, but it's fine to leave as it is.

Reply:
Maybe, I want to move it to a configuration file in the future.